boards/cats_rev1Pro/Core/Src/config/globals.c

```diff
@@ -46,6 +46,15 @@ const ICM20601 ICM3 = {
     .gyro_dps = ICM20601_GYRO_RANGE_2000_DPS,
```

Comment:
do we still have this guy?

Reply:
if so, why is it's cs_pin GPIO_PIN_2 and the other ones' pins are CS_IMUx_PIN

Reply:
Will create an issue to move the icm driver to new spi library